recipes/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.conf import settings
+
+class Book(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
+    name = models.CharField(max_length=100)
+
+class Ingredient(models.Model):
+    name = models.CharField(max_length=100)
+    note = models.CharField(max_length=1000)
```

Comment:
I have question about this ^ if length more than 256, maybe we can use TextField ?